### PR TITLE
Adding openshift plugin and a release config

### PR DESCRIPTION
**Pull request by JaydipGabani**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Releases
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Bump Version
+        default: v1.0.0
```
Comment (sseago) [inline .github/workflows/release.yml:7]: I'm not sure what release we should be at, but it's not 1.0.0 yet, which I don't think we'll hit until we're ready for GA.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        required: true
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (sseago) [inline .github/workflows/release.yml:11]: I'm not sure what this one should be, etiher, but ubuntu-latest doesn't seem like the right thing.

File: .github/workflows/release.yml (hunk 1 above)
Comment (JaydipGabani) [reply .github/workflows/release.yml:7]: +1, makes sense!

**Comment by JaydipGabani** [comment]
Moving plugin to its own repo

**Pull request closed without merging**